<launch>
    <node pkg="arx_r5_controller" type="R5Controller" name="arm" output="screen">
        <param name="arm_control_type" value="normal_v2"/>
        <param name="arm_can_id" value="can1"/>
        <param name="arm_end_type" value="0"/>
        
        <param name="sub_topic_name" value="/arm_cmd"/>
        <param name="pub_topic_name_v2" value="/arm_status"/>
    </node>
</launch>
